Error: RuntimeError: Out of bounds memory access (evaluating 'instance.exports._start()')
Stack: python.wasm.wasm-function[_PyErr_SetObject]@[wasm code]
python.wasm.wasm-function[PyErr_Format]@[wasm code]
python.wasm.wasm-function[_Py_module_getattro_impl]@[wasm code]
python.wasm.wasm-function[_Py_module_getattro]@[wasm code]
python.wasm.wasm-function[PyObject_GetAttr]@[wasm code]
python.wasm.wasm-function[_PyEval_EvalFrameDefault]@[wasm code]
python.wasm.wasm-function[_PyEval_Vector]@[wasm code]
python.wasm.wasm-function[_PyFunction_Vectorcall]@[wasm code]
python.wasm.wasm-function[_PyObject_CallFunctionVa]@[wasm code]
python.wasm.wasm-function[PyObject_CallMethod]@[wasm code]
python.wasm.wasm-function[_PyImport_InitCore]@[wasm code]
python.wasm.wasm-function[pycore_interp_init]@[wasm code]
python.wasm.wasm-function[pyinit_config]@[wasm code]
python.wasm.wasm-function[Py_InitializeFromConfig]@[wasm code]
python.wasm.wasm-function[pymain_main]@[wasm code]
python.wasm.wasm-function[Py_BytesMain]@[wasm code]
python.wasm.wasm-function[main]@[wasm code]
python.wasm.wasm-function[__main_void]@[wasm code]
python.wasm.wasm-function[_start]@[wasm code]
start@module-ready-wasi-py.js:498:32
@module-ready-wasi-py.js:5225:15
